packages/base64/package.json

```diff
@@ -32,8 +32,8 @@
   },
   "scripts": {
     "build": "exit 0",
-    "build:types": "tsc --build tsconfig.build.json",
-    "clean:types": "git clean -f '*.d.ts*'",
+    "prepack": "tsc --build tsconfig.build.json",
+    "postpack": "git clean -f '*.d.ts*'",
```

Comment:
Do we have a reason to believe that cleaning up types in `postpack` will work now when it did not before? https://github.com/endojs/endo/pull/1864

Reply:
Many things have changed since then. Named exports and imports for one.

To be sure, can you describe how I could reproduce the problem described in that PR intro? I'd like to try making a regression test, something that failed right before 1864 and passed right after.

Reply:
Integration with Agoric SDK satisfies me. Hopefully it works this time because this is by far the better state.